presentation/src/main/java/daily/dayo/presentation/view/Loading.kt

```diff
@@ -0,0 +1,83 @@
+package daily.dayo.presentation.view
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.LottieConstants
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+import daily.dayo.presentation.theme.Dark
+import daily.dayo.presentation.theme.DayoTheme
+import daily.dayo.presentation.theme.Gray7_F6F6F7
+
+const val LOADING_TOP_Z_INDEX = Float.MAX_VALUE
+
+@Composable
+fun Loading(
+    isVisible: Boolean,
+    modifier: Modifier = Modifier,
+    message: String = "잠시만 기다려 주세요",
+    dimColor: Color = Dark.copy(alpha = 0.4f),
+    lottieFile: Int,
```

Comment:
1. lottieFile이 필수 파라미터라면 파라미터 순서를 위쪽으로 올려주세요. 아니면 디폴트 값을 넣어둬도 될 것 같습니다! 
2. 아래 코드에서 rawResId로 사용되는 것 같은데 `@RawRes` lottieFile: Int 로 어노테이션을 붙이는 것이 좋아보입니다 

```suggestion
  @RawRes lottieFile: Int = R.raw.dayo_loading,
```

3. 그러면서 디자인에 맞게 기본 lottieModifier 크기도 92x85로 설정 되어있으면 더 편하게 쓸 수 있을 것 같아요!

Reply:
반영하였습니다 👍